src/main/java/org/broadinstitute/hellbender/tools/exome/TargetAnnotation.java

```diff
@@ -11,14 +11,21 @@
  * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
  */
 public enum TargetAnnotation {
-
     /**
      * Fraction of bases that are G or C in the reference overlapping the target.
      *
      * <p>This value is always a number between 0 and 1 except in very rare cases with small targets with all, undefined bases (N or X),
-     * in this case it value will be {@link Double#NaN NaN}</p>.
+     * in this case it value will be {@link Double#NaN NaN}.</p>
+     */
+    GC_CONTENT(TargetTableColumn.GC_CONTENT),
+
+    /**
+     * Number of baits overlapping with the target. Only relevant in the context of target-enriched (e.g. WES) sequencing data.
+     *
+     * <p>This value is never {@link Double#NaN} and is non-negative. It can be zero or fractional (if a bait is shared by more
+     * than a single target).</p>
```

Comment:
(as a safety measure, the sex genotyping tool masks all targets with bait count < 1 and produces a warning asking the user to reassess target/bait lists)